test/Microsoft.AspNet.Mvc.ModelBinding.Test/ModelBindingTestHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.Mvc.ModelBinding.Validation;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Test
+{
+    public static class ModelBindingTestHelper
```

Comment:
nah, I would hate inherited code here, a static class/methods  is/are much more useful, we can debate about the name.